### ISSUE TITLE
Mushroom Kingdom map issue

### ISSUE THREAD

**Issue by cpurules** [P0nk/Cosmic#136]
The NPCs for the first Mushroom Kingdom map (106020000 - Mushroom Forest Field) are in the wrong spots.

Cosmic:
![image](https://user-images.githubusercontent.com/52503242/191634252-c6e948a1-cf52-41b8-9f7b-fa16461cd446.png)

Expected:
![image](https://user-images.githubusercontent.com/52503242/191635063-c5000cd4-4939-474e-a7f2-52fa93b676d6.png)

These are all supposed to be further to the left (with the exception of the Head Security Officer, who should be close to the portal but is in fact all the way to the left of the map).  You can see that the graphics for Mushking are slightly broken because he is supposed to be seated in his throne to the left (foothold 40)

It appears to be an issue in the WZ files for this map.  The server XML/WZ files have different footholds assigned than in a clean Map.wz, which line up with where the characters are (mis)placed

I am happy to fix this, I think that just pulling a copy of this map from the clean v83 WZ and updating will suffice - but not sure what the process is for changes to the WZ/XML files is :-)

**Comment by Shahar6** [comment]
they did work correctly on my side without changes, so it must be something related to ur map.wz file and the server map.wz file, are they in sync?

**Comment by cpurules** [comment]
> they did work correctly on my side without changes, so it must be something related to ur map.wz file and the server map.wz file, are they in sync?

Yes, it is an issue in the wz files - I am using the ones provided by Cosmic, haven't done any wz edits.  The provided wz files have incorrect foothold mappings when compared to a clean v83 install.

For example, 106020000.img -> life -> 0 has id=1300007 (Scarrs).  In a clean v83 WZ, the foothold (fh) is 27.  In Cosmic, opening the same NPC entry on the map, it has a fh of 34.  You can see in this fh rendering that it explains the issue.

![image](https://user-images.githubusercontent.com/52503242/193298661-494e0c85-b334-4cc2-8f76-6f2a1a8241a7.png)

I know I can fix this for myself by just re-importing this map from the clean wz files, but my hope is to get it fixed in the _provided_ wz files, until a time where Cosmic fully supports the clean files.
